main.go

```diff
@@ -44,6 +44,7 @@ func main() {
 	httpAPIAddr := flag.String("httpapi-addr", defaultHTTPAPIAddr, "Specify an IP and port to run the HTTP API server on")
 	delegateAddr := flag.String("delegate-addr", "", "API endpoint for delegated routing")
 	delegateTimeout := flag.Int("delegate-timeout", 0, "Timeout for delegated routing in seconds")
```

Comment:
Can we change this to a safer default, like 10s or something?  We almost never want to disable timeouts entirely.

Reply:
Done on line 121 now where defaults are over-ridden by environment variables.